android/clientmanager/src/main/java/io/mosip/registration/clientmanager/repository/ApplicantValidDocRepository.java

```diff
@@ -18,11 +20,18 @@ public ApplicantValidDocRepository(ApplicantValidDocumentDao applicantValidDocum
     }
 
     public List<String> getDocumentTypes(String applicantType, String categoryCode, String langCode) {
-        if (applicantType == null)
-            return this.applicantValidDocumentDao.findAllDocTypesByDocCategory(categoryCode);
-
-        return this.applicantValidDocumentDao.findAllDocTypesByDocCategoryAndApplicantType(applicantType,
-                categoryCode);
+        List<String> nameList;
+        List<String> documentList = new ArrayList<>();
+        if (applicantType == null) {
+            nameList = this.applicantValidDocumentDao.findAllDocTypesByDocCategory(categoryCode);
+        }else {
+            nameList = this.applicantValidDocumentDao.findAllDocTypesByDocCategoryAndApplicantType(applicantType,
+                    categoryCode);
+        }
+        nameList.forEach((v) -> {
+            documentList.add(this.applicantValidDocumentDao.getDBValue(v,langCode).get(0));
```

Comment:
this maybe prone to nullpointer exception, we need to handle such cases

Reply:
handle the null pointer exception